falcon_multiqc/commands/connect.py

```diff
@@ -51,13 +51,14 @@ def cli(uri):
         while True:
             if not uri:
                 database = click.prompt("\nEnter database name to connect, or enter new name to create a new falcon_multiqc database")
-            if (database,) in databases or (uri[4],) in databases:     
+            if (database,) in databases or (uri and (uri[4],)) in databases:     
```

Comment:
mmm are you sure this is a bug? 
Because the 'databases' list contains a list of single string tuples like [(db1,), (db2,), (db3,)] but the uri string is something like 'postgres+psycopg2://USERNAME:PASSWORD@IP_ADDRESS:PORT/DATABASE_NAME', so I don't know whether you can check if that's in the databases list? the (uri[4],) in this case would be (DATABASE_NAME,)

Reply:
When I try to make a new database with connect, it has an error due to this line.

If URI is not given its null and checking its 4th index

Reply:
While I’m here what is the comma syntax in a python if statement? Haven’t seen it before

Reply:
oh whoa, yeah I got that too, okay nice!

Reply:
Umm the comma just means it's a tuple, to have a single item in a tuple you have to have at least 1 comma. So a 2 item tuple is (a, b), single item is (a,)

Reply:
If you just search uri[4] against the databases list, it won't find anything since it's a list of tuples of form (a,), so instead of changing the list, I put url[4] inside a tuple (uri[4],) and searched against it

Reply:
Oh hahahah yeah tuples, thanks for explaining!